### PR TITLE
add the ability to send message on enter or input action keyboard and fix layout issues

### LINKED ISSUE #52: Message not sent by hitting action keyboard
**Describe the bug**
We don't have any way to send the message by hitting the keyboard action, neither to configure it to "send" action. Would be very nice to be able to do it 

**To Reproduce**
click keyboard action, keyboard just close but message is not sent

**Expected behavior**
Message is sent

### LINKED ISSUE #60: Overflow exception
**Describe the bug**
There is an overflow exception when the chat doesn't take the full width of the screen

**To Reproduce**
Use chat dash in master detail view and send a very long message

**Expected behavior**
Not having overflow 

**Screenshots**
Capture d’écran 2020-04-21 à 10 43 07

### PR DESCRIPTION
**Pull request by jaumard**
fix #52
fix #60 by using layout builder instead of MediaQuery

### CODE REVIEW

**Comment by jaumard** [comment]
There will be no breaking changes as if the layout doesn't have constraint it will use MediaQuery like it does currently

File: lib/src/chat_view.dart [hunk 1]
```diff
@@ -420,120 +420,127 @@ class DashChatState extends State<DashChat> {
...
-                            quickReplyTextStyle: widget.quickReplyTextStyle,
-                          ),
-                        )
-                        .toList(),
-                ],
-              ),
+    return LayoutBuilder(
+      builder: (context, constraints) {
+        final maxWidth = constraints.maxWidth == double.infinity ? MediaQuery.of(context).size.width : constraints.maxWidth;
+        final maxHeight = constraints.maxWidth == double.infinity ? MediaQuery.of(context).size.height : constraints.maxWidth;
```
Comment (fayeed) [inline lib/src/chat_view.dart:426]: This should be `maxHeight`

Comment (fayeed) [reply lib/src/chat_view.dart:426]: Other than this everything looks good

Comment (jaumard) [reply lib/src/chat_view.dart]: Cool :) good catch ! I just push the fix :)

**Comment by fayeed** [comment]
Ok everything looks great I will merge these changes

**Comment by fayeed** [comment]
Thanks for your contribution.

**Pull request merged**